File: API/InvestmentPortfolio/History.cs
namespace InvestmentPortfolio
{
  public class History
  {
    public int ID { get; set; }
    public string Entry { get; set; } = string.Empty;
    public string Type { get; set; } = string.Empty;
    public DateTime Entry_Date { get; set; } = DateTime.Now;
  }
}

File: project/src/app/models/history.ts
export class History {
  public id: number;
  public entry: string;
  public type: string;
  public entry_Date: Date;

  constructor(id: number, type: string, entry: string, entry_Date: Date) {
    this.id = id;
    this.entry = entry;
    this.type = type;
    this.entry_Date = entry_Date;
  }
}